src/sonic_ax_impl/mibs/vendor/cisco/ciscoSwitchQosMIB.py

```diff
@@ -128,9 +132,11 @@ def update_stats(self):
             if_queues = self.port_queue_list_map[if_index]
             namespace = self.port_index_namespace[if_index]
 
-            # The first half of queue id is for ucast, and second half is for mcast
-            # To simulate vendor OID, we wrap queues by half distance
-            pq_count = math.ceil((max(if_queues) + 1) / 2)
+            # When create_only_config_db_buffers is set to true there are only ucast queues
+            # Otherwise the first half of queue id is for ucast, and second half is for mcast
+            # In that case to simulate vendor OID, we wrap queues by half distance
+            div = 1 if create_only_config_db_buffers == 'true' else 2
+            pq_count = math.ceil((max(if_queues) + 1) / div)
```

Comment:
It assumed
1. the number of counters equals the number of queues 
2. the number of UC queues equals the number of MC queues

After the change, assumption 2 is still true but assumption 1 can be false.
I do not suggest calculating `pq_count` by dividing `max(if_queues)` by 2 by default or 1 when `create_only_config_db_buffers` is configured. It can still break the logic in case a user configures a buffer on an MC queue.
I suggest the following ways to maintain backward compatibility after `create_only_config_db_buffers`
1. leveraging the mapping `COUNTERS_QUEUE_TYPE_MAP`
2. fetching the number of queues from `STATE_DB.BUFFER_MAX_PARAM_TABLE|<port>.max_queues` 

However, I have to say, this is still not 100% correct especially if a vendor doesn't have